components/_global-navigation.scss

```diff
@@ -10,10 +10,10 @@ $nav-height: 49px;
 	&__content-bar {
 		height: $nav-height;
 		margin: 0 auto;
-		width: 1024px;
+		width: 1004px;
```

Comment:
I wonder if we should have `width: 1024px - $scrollbar-width` just to make the logic behind that number obvious.